platform_versioned_docs/version-24.1/cloud/changelog.mdx

```diff
@@ -5,6 +5,12 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
+### v24.2.0_cycle23 Cloud - 28 August 2024
```

Comment:
We'll need to update the date here to the actual release date before we publish.